CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.61.0
+------------------------------
+*August 6, 2019*
+
+### Added
+- `c-pageBanner--loading` modifier to remove margin whilst images are loading.
+
+### Changed
+- Increased margin top for on images on wide screens for `c-pageBanner--narrow`.
```

Comment:
NIT – `Increased margin top for on images` > `Increased margin-top for images`

Reply:
Done.